wpiutil/src/main/java/edu/wpi/first/util/struct/DynamicStruct.java

```diff
@@ -628,5 +676,5 @@ private void setFieldImpl(StructFieldDescriptor field, long value, int arrIndex)
   }
 
   private final StructDescriptor m_desc;
-  private final ByteBuffer m_data;
+  final ByteBuffer m_data;
```

Comment:
What was the reason for this change?

Reply:
That would make sense, but it looks like the tests (or at least, that ones you added) don't (currently) access `m_data` directly- Instead they seem to use `setStringField` and `getStringField`.